lightning-background-processor/src/lib.rs

```diff
@@ -656,6 +672,9 @@ use futures_util::{dummy_waker, OptionalSelector, Selector, SelectorOutput};
 /// #     F: lightning::chain::Filter + Send + Sync + 'static,
 /// #     FE: lightning::chain::chaininterface::FeeEstimator + Send + Sync + 'static,
 /// #     UL: lightning::routing::utxo::UtxoLookup + Send + Sync + 'static,
+/// # 	  D: lightning::sign::ChangeDestinationSource + Send + Sync + 'static,
```

Comment:
I don't know if it is worth keeping this example. Another thing to update besides ldk-node and ldk-sample.